roles/openshift_excluder/molecule.yml

```diff
@@ -0,0 +1,40 @@
+---
+# http://molecule.readthedocs.io/en/stable-1.22/usage.html
+molecule:
+  test:
+    sequence:
+      - destroy
+      - create
+      - converge
+      - verify
+# pip install pytest-cov
+# https://stackoverflow.com/questions/26589990/py-test-error-unrecognized-arguments-cov-ner-brands-cov-report-term-missi
+dependency:
+  name: galaxy
+driver:
+  name: docker
+docker:
+  containers:
+    - name: openshift_excluder_disable
+      image: rhel7-ansible
+      image_version: latest
+      ansible_groups:
+        - openshift_excluder_disable
+      dockerfile: Dockerfile
+    - name: openshift_excluder_enable
+      image: rhel7-ansible
+      image_version: latest
+      ansible_groups:
+        - openshift_excluder_enable
+      dockerfile: Dockerfile
+ansible:
+  config_file: ./ansible.cfg
+  extra-vars: >
```

Comment:
This will get populated with ansible variables (one combination per each generated `molecule.yml`)